### ISSUE TITLE
Insecta Mundi - Batch processing fonts

### ISSUE THREAD

**Issue by flsimoes** [gsautter/goldengate-imagine#905]
I've found quite a few issues discussing font issues when batch processing, the main one being [#390](https://github.com/gsautter/goldengate-imagine/issues/390), so I wasn't sure on where to discuss this one.

As I understand it, this is generally sorted out through changing the the `FM` and `CS` patterns (I've used `FM=D` and `CS=S` in this case).

How should I deal with situations such as this? From what I read in [#348](https://github.com/gsautter/goldengate-imagine/issues/348), changing the patterns is used more for when a whole set of characters are being misinterpreted, whereas in the case of Insecta Mundi it is mostly restricted to one character.
Several "V" are interpreted as a latinized "V" or "U" (not sure how to name that character). In the example below, we should get "Oreodera vulgata".

![image](https://user-images.githubusercontent.com/3343086/74662282-64f62600-5178-11ea-934e-5c59ef3ce3ca.png)

**Comment by gsautter** [comment]
Have you tried `FM=U`? Also, how does this character come out if you copy & paste the respective text from Acrobat to a text editor? And ultimately, what _should_ this character be?

**Comment by flsimoes** [comment]
Will try `FM=U`, will comment back when I see the result.

It comes out as "v" when I copy and paste it.

It should be a "v" (in this case, we would have "vulgata").

**Comment by gsautter** [comment]
So basically we have a lower case "v" that comes up as an upper case "V"?

**Comment by flsimoes** [comment]
Yes, but in a different font type (unless I'm completely misunderstanding how fonts are treated).

**Comment by flsimoes** [comment]
`FM=U` sorted it!

**Comment by gsautter** [comment]
If you know the journal doesn't do anything hinky with the fonts (the copy & paste test from Acrobat is a good litmus test here), `FM=U` is the fastest option, and also the safest if the font face is a bit exotic (different from the comparison baseline fonts).

What did you do the first run with anyway?

**Comment by flsimoes** [comment]
Thanks Guido!

I used `FM=D`, which I believe is the default.

**Comment by gsautter** [comment]
`FM=D` is the full broadside ... shuns any existing Unicode mapping and does it all itself ... the tool of choice for breaking obfuscating Unicode mappings (the stuff producing garble on the copy & paste test), but over the top for most other cases.

The main reason for it being the default nonetheless is that otherwise a user might end up with that total garble on first contact, which, to be honest, would drive me away when trying out a new tool.

**Comment by flsimoes** [comment]
It does make a lot of sense, I would be driven away as well. Thanks for the explanation!
